server/index.js

```diff
@@ -5,7 +5,7 @@ const port = 3000;
 const app = express();
 app.use(parser.json());
 
-app.use(express.static(__dirname + '/../dist'))
+app.use(express.static(__dirname + '/../dist'));
 //get the names and pics of the products
 app.get('/products', function (req, res) {
```

Comment:
Do you need this route? Will you ever need to retrieve all your products at once?